products/opensuse/main.pm

```diff
@@ -38,6 +38,10 @@ sub is_leap {
     return get_var('VERSION') =~ /^42:S/;
 }
 
+sub is_leap_42_3_or_newer {
+    return is_leap && check_var('VERSION', '42.3');
+}
+
```

Comment:
Right, `lib/utils` might be even better home than `main_common`. I think the function should be modeled similar to `sle_version_at_least`. If we don't know what version - in Leap series - is after 42.3 (be it 42.4 or 15.0), I suggest we don't advertise, that we `return` any knowledge about it.